pkg/api/plugins.go

```diff
@@ -257,11 +257,12 @@ func (hs *HTTPServer) CollectPluginMetrics(c *models.ReqContext) response.Respon
 // GetPluginAssets returns public plugin assets (images, JS, etc.)
 //
 // /public/plugins/:pluginId/*
-func (hs *HTTPServer) GetPluginAssets(c *models.ReqContext) response.Response {
+func (hs *HTTPServer) GetPluginAssets(c *models.ReqContext) {
 	pluginID := c.Params("pluginId")
 	plugin := hs.PluginManager.GetPlugin(pluginID)
 	if plugin == nil {
-		return response.Error(404, "Plugin not found", nil)
+		c.Handle(hs.Cfg, 404, "Plugin not found", nil)
```

Comment:
When resumed, think you can use `c.JsonApiErr(404, "Plugin not found", nil)` rather than above.

Reply:
Yeah I just caught this now. Never seen it before 👀